contrib/mod_tls.c

```diff
@@ -11984,7 +11984,10 @@ MODRET set_tlsciphersuite(cmd_rec *cmd) {
   c = add_config_param(cmd->argv[0], 1, NULL);
 
   /* Make sure that EXPORT ciphers cannot be used, per Bug#4163. */
-  c->argv[0] = pstrcat(c->pool, "!EXPORT:", ciphersuite, NULL);
+  /* This breaks system profiles though, so don't change them.   */
+  if (strncmp(ciphersuite, "PROFILE=", 8)) {
+    c->argv[0] = pstrcat(c->pool, "!EXPORT:", ciphersuite, NULL);
+  }
```

Comment:
Hmm.  I expect that OpenSSL's support/handling of `PROFILE` depends on the _version_ of OpenSSL used, and/or possibly _how_ that OpenSSL installation is built/compiled?  I'd like to restrict the scope of this change to only the OpenSSL installations for which it is pertinent...